### DISCUSSION TITLE
Not working right for relaying anymore

### DISCUSSION THREAD

**Discussion by ChrisYHUP** [AzuraCast/AzuraCast#5771] [Q&A] [answered]
Hi,

I've been using Azuracast for about a year-ish. I never used the AutoDJ or anything like that, I only used it so I could relay to it from my SamCloud broadcaster, and be able to log the songs played for royalty reasons. It worked flawlessly, broadcasting to Shoutcast2, logging everything. But since updating and setting it up on a new domain, I found that I couldn't even start the radio without adding a file to the AutoDJ, and when I finally managed to start it and relay to it, it doesn't actually show the songs being played. I can see what is being played on the shoutcast DNAS

https://radio.yhup.net:8005/index.html?sid=1

But it does not show in the Azuracast UI and the player says it is offline.

https://radio.yhup.net/public/yhup

Also my admin page says it doesn't send any data..

https://radio.yhup.net:8005/admin.cgi?sid=1

**Comment by ItsMitchh** [comment, marked as answer]
I looked at the URLs you provided, looks like the now playing data is working as intended. 

E: After listening to the streams, there is indeed nothing playing, the console reports error 404. Can you verify your streams are actually up and running properly?

**Reply by ChrisYHUP** [reply]
The now playing data started working. But I still cannot access the 'listener' URL in HTTPS so, as an example:

https://radio.yhup.net/radio/8005

That shows me 404 not found, yet when I use it on my website player it works?

If I try to do it like this,

https://radio.yhup.net:8005/radio.mp3

It says I am not authorised. The thing is I want to be able to provide a SSL url for apps and websites that I am going to be resubmitting the radio to. It works fine in HTTP

http://radio.yhup.net:8005/radio.mp3

EDIT: It works now.... Must have been cached issues? My head hurts. Thank you anyway!

**Comment by gmantzak** [comment]
Hi @ChrisYHUP , I wanted to ask you please if you relay from Sam Cloud to Azura or you use remote playlist. 
I have a similar setup where i would like to put on the front Azuracast to manage also drops and have more flexibility with the stream
Thank you
